src/execute_request.jl

```diff
@@ -5,54 +5,111 @@
 import Base.Libc: flush_cstdio
 import Pkg
 
-const text_plain = MIME("text/plain")
-const image_svg = MIME("image/svg+xml")
-const image_png = MIME("image/png")
-const image_jpeg = MIME("image/jpeg")
-const text_markdown = MIME("text/markdown")
-const text_html = MIME("text/html")
-const text_latex = MIME("text/latex") # Jupyter expects this
-const text_latex2 = MIME("application/x-latex") # but this is more standard?
-const application_vnd_vega_v3 = MIME("application/vnd.vega.v3+json")
-const application_vnd_vegalite_v2 = MIME("application/vnd.vegalite.v2+json")
-const application_vnd_dataresource = MIME("application/vnd.dataresource+json")
+Base.showable(a::Vector{MIME}, x) = any(m -> showable(m, x), a)
+
+"""
+A vector of MIME types (or vectors of MIME types) that IJulia will try to
+render. IJulia will try to render every MIME type specified in the first level
+of the vector. If a vector of MIME types is specified, IJulia will include only
+the first MIME type that is renderable (this allows for the expression of
+priority and exclusion of redundant data).
+
+For example, since "text/plain" is specified as a first-child of the array,
+IJulia will always try to include a "text/plain" representation of anything that
+is displayed. Since markdown and latex are specified within a sub-vector, IJulia
+will always try to render "text/markdown", and will only try to render
+"text/latex" if markdown isn't possible.
+"""
+const ijulia_mime_types = Vector{Union{MIME, Vector{MIME}}}([
+    MIME("text/plain"),
+    MIME("image/svg+xml"),
+    MIME("image/png"),
+    MIME("image/jpeg"),
+    MIME("text/html"),
+    [
```

Comment:
To be the same behavior as before, text/html should be in the list after text/markdown